assets/components/organisms/media-mondrian.hbs

```diff
@@ -0,0 +1,251 @@
+      </div>
+      <div class="clearfix"></div>
+    </div>
+  </a>
+</div>
+  <div class="media-mondrian-item media-mondrian-item-sm">
+    {{> axis }}
+  </div>
+
+  <div class="media-mondrian-item media-mondrian-item-xs">
+    <a href="detail.html">
+      <div class="media-mondrian-thumbnail">
+        <img src="http://notrehistoire.ch.s3-eu-west-1.amazonaws.com/photos/2016/01/ba610aba0e3d3701_JPG_464x464_upscale_q85.jpg" class="img-responsive" alt="" width="200"/>
+      </div>
+      <div class="media-mondrian-caption">
+        <span>David et Goliath 1959</span>
+        <div class="pull-right">
+            <small><i class="glyphicon glyphicon-picture"></i></small>
+        </div>
+        <div class="clearfix"></div>
+      </div>
+    </a>
+  </div>
+
+  <div class="media-mondrian-item media-mondrian-item-xs">
+    <a href="detail.html">
+      <div class="media-mondrian-thumbnail">
+        <img src="http://www.ticketcorner.ch/obj/media/CH-eventim/galery/222x222/m/montreux-jazz-festival-tickets.gif" class="img-responsive" alt="" width="200" height="200"/>
+      </div>
+      <div class="media-mondrian-caption">
+        <span>Fond d'archive du Montreux Jazz</span>
+        <div class="pull-right">
+            832&nbsp;<small><i class="glyphicon glyphicon-file"></i></small>
+        </div>
+        <div class="clearfix"></div>
+      </div>
+    </a>
+  </div>
+  <div class="media-mondrian-item media-mondrian-item-comment media-mondrian-item-sm">
+    {{> comment }}
+    <div class="media-mondrian-caption">
+      <span>Le bonheur parmi les narcisses!</span>
+      <div class="pull-right">
+          5&nbsp;<small><i class="glyphicon glyphicon-comment"></i></small>
+      </div>
+      <div class="clearfix"></div>
+    </div>
+  </div>
+  <div class="media-mondrian-item media-mondrian-item-sm">
+    <a href="detail.html">
+      <div class="media-mondrian-thumbnail">
+        <img src="http://notrehistoire.ch.s3-eu-west-1.amazonaws.com/photos/2014/07/e29c33b5154da63f_jpg_530x530_q85.jpg" class="img-responsive" alt="" width="400"/>
+      </div>
+      <div class="media-mondrian-caption">
+        <span>Faire schmolitz!</span>
+        <div class="pull-right">
+            <small><i class="glyphicon glyphicon-picture"></i></small>
+        </div>
+        <div class="clearfix"></div>
+      </div>
+    </a>
+  </div>
+  <div class="media-mondrian-item media-mondrian-item-xs">
+    <a href="detail.html">
+      <div class="media-mondrian-thumbnail">
+        <img src="http://notrehistoire.ch.s3-eu-west-1.amazonaws.com/users/avatar/2010/04/d1956651118ec894_png_168x168_crop_q85.jpg" class="img-responsive" alt="" width="200"/>
+      </div>
+      <div class="media-mondrian-caption">
+        <span>Raphael Briner</span>
+        <div class="pull-right">
+            <small><i class="glyphicon glyphicon-user"></i></small>
+        </div>
+        <div class="clearfix"></div>
+      </div>
+    </a>
+  </div>
+
+</div>
```

Comment:
@zufrieden Not sure you if you understand the principle of modularity 😅  What's the point of making `thumbnail-mondrian` and `media-mondrian-item` if you're not using it ?